src/Sulu/Bundle/MediaBundle/Media/FormatCache/FormatCacheClearer.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\MediaBundle\Media\FormatCache;
+
+/**
+ * Format cache clearer.
+ */
+class FormatCacheClearer
```

Comment:
For the multi cache support it would be good instead of use a new service `format_cache_clearer` add the clear function to `format_manager` removes the `format_cache` service there from the construct and add a simple string `defaultCache` which is `local`.